Chaincase.Common/Global.cs

```diff
@@ -123,7 +110,7 @@ public async Task InitializeNoWalletAsync(CancellationToken cancellationToken =
 
                 Logger.LogDebug($"Global.InitializeNoWalletAsync(): Got lock");
 
-                Cache = new MemoryCache(new MemoryCacheOptions
+                MemoryCache Cache = new MemoryCache(new MemoryCacheOptions
```

Comment:
this can also be in the DI. I think .net core provides some direct extensions for this (exposed as IMemoryCache)

Reply:
If we don't use it anywhere else do we want it in the DI? It could just be local to the lifecycle. Why complicate the DI?